### PR TITLE
Add relevant new info to Symfony topic

### PR DESCRIPTION
**Pull request by alelazcano**
### Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: <https://github.com/github/explore/blob/main/CONTRIBUTING.md>.
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [x] Suggesting edits to an existing topic or collection
  - [ ] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288, size <= 75 kB)
- [x] Content (and my changes are in `index.md`)

> Add relevant new info to Symfony topic, before 6y ago last commit.

### CODE REVIEW

**Review by kenyonj — CHANGES_REQUESTED** [review]

File: topics/symfony/index.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
-aliases: symfony2, symfony3, symfony-component, symfony-bundle
+aliases: symfony2, symfony3, symfony4,symfony5,symfony6,symfony-framework,symfony-component, symfony-bundle
+related: prestashop, laravel, cakephp, codeigniter, php, php-framework, php-component, php-library, php-package
```
[the remark this answers is not in the document]
Comment (Budgurl) [reply topics/symfony/index.md:3]: There are some other ways

**Comment by shahin7071** [comment]
> `

**Comment by shahin7071** [comment]
12

**Comment by ahpook** [comment]
Closing this one as being spammy - it plagiarizes the copy from the website and has too many unrelated topics.

**Pull request closed without merging**